open-api/rest-catalog-open-api.yaml

```diff
@@ -4817,6 +4955,16 @@ components:
           }
         }
 
+    IdempotencyInProgressError:
+      summary: A request with the same Idempotency-Key is already in progress
+      value: {
+          "error": {
+            "message": "A request with this Idempotency-Key is still in progress",
+            "type": "RequestInProgressException",
```

Comment:
This fragment is an example from the Open API perspective. Is the `type` code normative or informational?